SCMS.Services.Api/Models/Foundations/AdditionalDetails/AdditionalDetail.cs

```diff
@@ -0,0 +1,31 @@
+﻿// -----------------------------------------------------------------------
```

Comment:
Remove empty new lines.
Rename 'Fide' to 'FideId'
Group Created By and Updated By together
Group Created Date and Updated Date together

Reply:
I was able to fix this